wger/manager/tests/test_weight_log.py

```diff
@@ -160,7 +160,7 @@ class WeightLogOverviewAddTestCase(WorkoutManagerTestCase):
     Tests the weight log functionality
     '''
 
-    def add_weight_log(self, fail=True):
+    def add_weight_log(self, fail=True, rel=False):
```

Comment:
What is the use of `rel=False` here?

Reply:
I was supposed to use it to check the relationship in my test but it is no longer necessary. I have since removed it.